src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.io.*;
+import java.util.Scanner;
+import duke.Deadline;
+import duke.Task;
+import duke.Event;
+import duke.Todo;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class Duke {
+
+    static final int BY_AT_INDEX = 4;
+    static final int FIND_INDEX = 5;
+    static final int TODO_INDEX = 5;
+    static final int EVENT_INDEX = 6;
+    static final int DELETE_INDEX = 7;
+    static final int DEADLINE_INDEX = 9;
+    static final int MAX_NO_OF_TASKS = 100;
```

Comment:
This is good if you impose file saving limits. However, do consider how you plan to handle the 101th task.